### ISSUE TITLE
Add a link to thredds endpoint on gliderDAC homepage

### ISSUE THREAD

**Issue by Bobfrat** [ioos/ioos-us#171]
Glider Data access still baffles people. Need to add a THREDDS link to the Data Visualization and Access menu on gliders.ioos.us/data

Thredds URL:
https://data.ioos.us/gliders/thredds/catalog.html

**Comment by gitchrisadams** [comment]
@Bobfrat 
What do you think abou this?
![image](https://user-images.githubusercontent.com/9978238/37913641-5f399a42-30e3-11e8-9114-d9d7dbc5686b.png)

Or full Thredds def from https://www.unidata.ucar.edu/software/thredds/current/tds/TDS.html
or other description?
The THREDDS Data Server (TDS) is a web server that provides metadata and data access for scientific datasets, using OPeNDAP, OGC WMS and WCS, HTTP, and other remote data access protocols. The TDS is developed and supported by Unidata, a division of the University Corporation for Atmospheric Research (UCAR), and is sponsored by the National Science Foundation.

**Comment by Bobfrat** [comment]
I think we can be more specific to GliderDAC here;

Can you put it under ERDDAP Catalog?


**THREDDS Catalog**
The THREDDS Data Server (TDS) is a web server that provides metadata and data access to all GliderDAC datasets, using a variety of remote data access protocols.

**Comment by gitchrisadams** [comment]
Yes will put under ERDDAP and update to include that description. Thanks.
![image](https://user-images.githubusercontent.com/9978238/37914189-a2ba51b6-30e4-11e8-8d46-e103cbc40e5d.png)

**Comment by gitchrisadams** [comment]
Created PR: https://github.com/ioos/gliders/pull/50
